app/views/orders/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class='container'>
+  <h1>Order no. <%= @order.id %> details</h1>
+  <h3>Created at: <%= @order.created_at %></h3>
+  <h3>Current status: <span class="label label-default"><%= @order.status %></span>
+    <% if @order.completed? || @order.cancelled? %>
+      on <%= @order.updated_at %>
+    <% end %>
+  </h3>
+
+  <table class = "table">
+    <tr>
+      <th>Outing</th>
+      <th>Quantity</th>
+      <th>Sub Total</th>
+    </tr>
+    <% @order.order_items.each do |order_item|%>
+      <tr>
+        <td><%= link_to order_item.outing.title, outing_path(order_item.outing) %></td>
+      </tr>
+      <tr>
+        <td><%= order_item.quantity %></td>
+      </tr>
+      <tr>
+        <td><%= order_item.subtotal %></td>
+      </tr>
+    <% end %>
+  </table>
+</div>
```

Comment:
@drod1000 @Dpalazzari @njgheorghita Quick update: again very good catch of missing the total from the page. You guys just highlighted a weakness in the test cases. Since the order only had a single outing, even if its quantity was more than one, the `subtotal` was essentially the same as `total` and that is why the test passed. I am working on making the test cases stronger to ensure we only pass if total is display and the correct `order.total` is displayed. Thank you again for highlighting this!!!